package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-geosuggest",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
Please do not release in feature branches! The release will be done in the master branch and may have more than one PR in it.